Server/mods/deathmatch/logic/CGame.cpp

```diff
@@ -3692,6 +3693,23 @@ void CGame::Packet_Vehicle_InOut(CVehicleInOutPacket& Packet)
                                         CVehicleInOutPacket JackedReply(pJacked->GetID(), VehicleID, 0, VEHICLE_NOTIFY_OUT_RETURN);
                                         m_pPlayerManager->BroadcastOnlyJoined(JackedReply);
 
+                                        CLuaArguments Arguments;
+                                        Arguments.PushElement(pJacked);                  // player / ped
```

Comment:
Since `onPlayerVehicleExit` & `onPedVehicleExit` are being called first, this arg should be `pVehicle`

Sorry, I got confused as well, which is why I edited my last comment.